### PR TITLE
Add `langsmith`

### PR DESCRIPTION
**Pull request by hinthornw**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vowelparrot** [comment]
(I am willing to be included here)

File: recipes/langsmith/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  run:
+    - python >=3.8.1,<4.0
+    - pydantic >=1.0.0,<2.0.0
+    - requests >=2.0.0,<3.0.0
+    - tenacity >=8.1.0,<9.0.0
+
+test:
+  imports:
+    - langsmith
+  commands:
```
[the remark this answers is not in the document]
Comment (hinthornw) [reply recipes/langsmith/meta.yaml:35]: Thanks, @synapticarbors ! Also added a couple more imports. Is there anything else I ought to test or include?

File: recipes/langsmith/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{% set name = "langsmith" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langsmith-{{ version }}.tar.gz
+  sha256: cf317b47c506f8c89a3b3a932d6fbcfb45ae1f43ed555d23b71d166ab9db21cf
```
Comment (synapticarbors) [inline recipes/langsmith/meta.yaml:10]: looks like when you updated the version, you didn't update the sha256.

Comment (hinthornw) [reply recipes/langsmith/meta.yaml:10]: Got it! I'll update. Thank you for your help :)

**Comment by hinthornw** [comment]
Thank you!

**Pull request merged**